BUILD.md

```diff
@@ -19,15 +19,15 @@
 
 Daffodil's build requirements include:
 
-* JDK 8 or higher
-* SBT 0.13.8 or higher
+* Java 8 or higher
+* sbt 0.13.8 or higher
 * C compiler C99 or higher
 * Mini-XML Version 3.0 or higher
 
-You will need the Java Software Development Kit ([JDK]) and the Scala
-Build Tool ([SBT]) to build Daffodil, run all tests, create packages,
-and more.  You can install the latest [Java 11 LTS][JDK] version and
-the latest [SBT] version following their websites' instructions or
+You will need the Java Software Development Kit ([Java]) and the Scala
+Build Tool ([sbt]) to build Daffodil, run all tests, create packages,
+and more.  You can install the latest [Java 11 LTS][Java] version and
```

Comment:
Why stipulate Java 11? 
We should either say Java 8, or Java 8 (Java 11 and Java 17 are also supported)